src/site/_includes/components/YouTube.js

```diff
@@ -29,6 +29,13 @@ module.exports = (id, startTime) => {
     throw new Error("Cannot create YouTube component if id is undefined");
   }
 
+  // Don't load YT iframe in our test environment. This specifically affects
+  // screenshot testing where the iframe can be slow or flaky to load and fail
+  // because YouTube is always fiddling with their UI.
+  if (process.env.ELEVENTY_ENV === "test") {
```

Comment:
`.w-youtube` is still going to create a box of a certain w/h (with the `padding` ratio hack). Is it worth including the physical box and giving it a background color so it looks like a red/black square?

Reply:
I don't think it will because since we're returning early the shortcode never writes out `<div class="w-youtube">`. It just writes out a blank text node.

Reply:
My point is that we _could_ write out that div, just not include the iframe, so the layout is more akin to the real one if anyone is actually looking at the visual diffs.

I could perhaps accidentally add 100s of YouTube videos and I would not be alerted to that fact. If it's a giant red box of the right size, at least I'd know.